docs/sphinx/user_guide/using_raja.rst

```diff
@@ -18,6 +18,20 @@ are visible, and linking against the RAJA library. We maintain a
 shows how to use RAJA in a CMake project, either as a Git submodule or
 as an externally installed library that you link your application against.
 
+It is important to ensure that you pass in compiler options in your project
+that match the options that you compiled RAJA with. Specifically, if you built
+RAJA with `-DENABLE_OPENMP=On` then you need to add a flag like `-fopenmp` when
```

Comment:
I would change this to 'you need to pass to the compiler the OpenMP flag appropriate for the compiler, such as `-fopenmp`.